std/json.d

```diff
@@ -230,6 +230,43 @@ struct JSONValue
         return store.floating;
     }
 
+    /***
+     * Value getter/setter for boolean stored in JSON
+     * Throws: `JSONException` for read access if `this.type` is not
+     * `JSONType.true_` or `false_`.
```

Comment:
Prefer spelling out the full name:
```
`JSONType.true_` or `JSONType.false_`.
```

Reply:
Done. I find the comment has too much implementation details but it's the same in the rest of the module.